layouts/default.vue

```diff
@@ -161,6 +170,12 @@ export default Vue.extend({
       } else if (!this.$store.state.page.showNavDrawer) {
         this.showNavDrawer({ showNavDrawer: true });
       }
+    },
+    searchQuery(newQuery: string, oldQuery: string): void {
+      // If neither the old nor the new query are empty, we don't want to move.
+      if (newQuery !== '' && oldQuery === '') {
```

Comment:
We might want to trim this so writing spaces doesn't trigger the push either?

Reply:
I agree

Reply:
Added